src/pages/FrontendMentor/YxZkCountdown/index.tsx

```diff
@@ -0,0 +1,74 @@
+import React, { useState, useEffect } from 'react'
+import TimeCard from '../../../components/YxZkCountdown/TimeCard'
+import BackgroundStars from '../../../assets/YxZkCountdown/bg-stars.svg'
+import BackgroundHills from '../../../assets/YxZkCountdown/pattern-hills.svg'
+import {
+  BackgroundContainer,
+  StarBackground,
+  HillBackground,
+  Header,
+  StarBackgroundContainer,
+  HillBackgroundContainer,
+  Countdown,
+  TimeCardContainer,
+  SocialMediaContainer,
+  SocialMediaIcon,
+} from './styles/YxZkCountdown.styled'
+
+import PinterestIcon from '../../../assets/YxZkCountdown/icon-pinterest.svg'
+import FacebookIcon from '../../../assets/YxZkCountdown/icon-facebook.svg'
+import InstagramIcon from '../../../assets/YxZkCountdown/icon-instagram.svg'
```

Comment:
consider rearranging the imports. You can place all the svg imports at the bottom